lib/iris/coords.py

```diff
@@ -1611,6 +1611,97 @@ def __hash__(self):
         return hash(id(self))
 
 
+class CellMeasure(AuxCoord):
+    """
+    A CF Cell Measure, providing area or volume properties of a cell
```

Comment:
As Iris itself makes no interpretation of the meaning of the 'measure' identity, I'm not sure we should be testing specifically for area/volume, or encoding any restriction on the name.
Likewise, here we can just say "providing a measure property of cells such as 'area' or 'volume'", and leave it at that.
As far as CF is concerned, a file can validly **contain** other measures, we just don't yet have any for which CF defines a specific meaning.